<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>consetting</class>
 <widget class="QDialog" name="consetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>314</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="lan_groupbox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>90</y>
     <width>221</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Connection Settings</string>
   </property>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>IP Address</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_25">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Port</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ip_lineedit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>151</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>192.168.1.72</string>
    </property>
    <property name="maxLength">
     <number>15</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="port_spinBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>151</width>
      <height>36</height>
     </rect>
    </property>
    <property name="maximum">
     <number>4000</number>
    </property>
    <property name="value">
     <number>3000</number>
    </property>
   </widget>
  </widget>
  <widget class="QRadioButton" name="usb_radio">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>USB Connection</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="lan_radio">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>LAN Connection</string>
   </property>
  </widget>
  <widget class="QPushButton" name="conform_close">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>270</y>
     <width>92</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>270</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>See Readme For LAN Info</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
